python/lsst/ctrl/oods/gen3ButlerBroker.py

```diff
@@ -0,0 +1,260 @@
+# This file is part of ctrl_oods
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import collections
+import logging
+import os
+import shutil
+from lsst.ctrl.oods.butlerBroker import ButlerBroker
+from lsst.ctrl.oods.timeInterval import TimeInterval
+from lsst.ctrl.oods.imageData import ImageData
+from lsst.ctrl.oods.archiverName import ArchiverName
+from lsst.daf.butler import Butler
+from lsst.obs.base.ingest import RawIngestTask, RawIngestConfig
+from lsst.obs.base.utils import getInstrument
+from astropy.time import Time
+from astropy.time import TimeDelta
+import astropy.units as u
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Gen3ButlerBroker(ButlerBroker):
+    """Processes files on behalf of a Gen3 Butler.
+
+    Parameters
+    ----------
+    config: `dict`
+        configuration of this butler ingester
+    """
```

Comment:
Parameters `publisher` and `publisher_queue` are not described in the docstring.

Reply:
Fixed